nss-day-cohort-51/holiday-road-grand-canyon-gliders
# added hover effects on buttons, smoothed transitions on footer cards, styled dropdowns

# worked on main.css to update styling

Added new logo, hover effects on buttons, smoothed transitions on footer cards, styled dropdowns

## Type of change



# Testing Instructions

Hover over all the buttons in the trip preview box including SAVE TRIP button to see if the hover effect works.  Same for our linked in cards in the footer. I added the ms to transition so now it doesn't look as clunky on hover.

# Checklist:
